scripts/rf_logs.py

```diff
@@ -58,5 +60,8 @@
         # Print log was requested
         log_entries = redfish_utilities.get_log_entries( redfish_obj, container_type, container_id, args.log )
         redfish_utilities.print_log_entries( log_entries, args.details )
+except BrokenPipeError:
+    pass
 finally:
     redfish_obj.logout()
+    sys.stderr.close()
```

Comment:
Can this be brought into the lower level print routine? I'd rather not create a burden on callers doing this since the "redfish_utilities" folder is supposed to be treated as a library others might use.

Reply:
I moved the catch of BrokenPipeError to print_log_entries(). But I left the `sys.stderr.close()` here because that seemed to be the only way to eliminate this extraneous message when piping to head:
```
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>
BrokenPipeError: [Errno 32] Broken pipe
```